lib/SILOptimizer/Differentiation/LinearMapInfo.cpp

```diff
@@ -537,10 +537,6 @@ bool LinearMapInfo::shouldDifferentiateInstruction(SILInstruction *inst) {
         isa<EndBorrowInst>(inst) || isa<DeallocationInst>(inst) ||
         isa<DestroyValueInst>(inst) || isa<DestroyAddrInst>(inst))
       return true;
-    // Should differentiate any instruction that creates an SSA copy of an
-    // active operand.
-    if (isa<CopyValueInst>(inst))
-      return true;
```

Comment:
Note: I tried to track down the origin of this code. I found it was added by [this commit](https://github.com/apple/swift/commit/607228c63dc4d6c721e626a0dfaf796b2c028f64), though it existed in a different form prior to the commit.

I think this special-case for `copy_value` was just an oversight/latent error.